Update warning message about enhancement on the repo (#442)
meilisearch/docs-scraper: README.md

*Our team is small, and our availability will be reduced in the upcoming times. As such, response times might be slower, and we will not be accepting enhancements for this repository moving forward.*

> *If you're looking for reliable alternatives, consider using [Cloud Service](https://www.meilisearch.com/pricing?utm_campaign=oss&utm_source=integration&utm_medium=meilisearch-migration). It offers a robust solution for those seeking an alternative to this repository, and also features a crawler for your convenience.*

Reviewer: ```suggestion
*If you're looking for reliable alternatives, consider using [Cloud Service](https://www.meilisearch.com/pricing?utm_campaign=oss&utm_source=integration&utm_medium=meilisearch-migration). It offers a robust solution for those seeking an alternative to this repository by providing a crawler for your convenience.*
```

Reviewer: Why "and also"? For me the alternative is the crawler, right?

Author: I think you're right, I was thinking of an alternative to meilisearch for the Cloud but this is much better as you write.
